### PR TITLE
Backport Zoom out: fix scaling issues (#65998) to WP 6.7

### PR DESCRIPTION
**Pull request by getdave**
Backport of https://github.com/WordPress/gutenberg/pull/65998

### CODE REVIEW

File: packages/block-editor/src/components/iframe/style.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	height: 100%;
+}
+
+.block-editor-iframe__scale-container.is-zoomed-out {
+	$container-width: var(--wp-block-editor-iframe-zoom-out-container-width, 100vw);
+	$prev-container-width: var(--wp-block-editor-iframe-zoom-out-prev-container-width, 100vw);
+	width: $prev-container-width;
+	// This is to offset the movement of the iframe when we open sidebars
+	margin-left: calc(-1 * (#{$prev-container-width} - #{$container-width}) / 2);
+}
```
Comment (getdave) [inline packages/block-editor/src/components/iframe/style.scss:17]: @MaggieCabrera This file appears to have been deleted on `wp/6.7` branch. But your PR includes it. Do you know what's going on there?

Comment (MaggieCabrera) [reply packages/block-editor/src/components/iframe/style.scss:17]: I think the problem is we are missing this PR https://github.com/WordPress/gutenberg/pull/65977

Comment (kevin940726) [reply packages/block-editor/src/components/iframe/style.scss:17]: Do we want to backport #65977 too?

Comment (getdave) [reply packages/block-editor/src/components/iframe/style.scss:17]: I don't know. Asked there. The dependencies here are getting really hard to follow what with work moving forward in Gutenberg Plugin.

Comment (ajlende) [reply packages/block-editor/src/components/iframe/style.scss:17]: While #65977 isn't strictly necessary for 6.7, the risks from backporting are low and bugfixes like this are being made in the CSS that was moved. So I'd say we should backport it just to make this one easier.

**Review by kevin940726 — APPROVED** [review]
